pom.xml

```diff
@@ -82,6 +82,24 @@
          </configuration>
        </plugin>
      </plugins>
+    <pluginManagement>
+      <plugins>
+      <plugin>
+        <groupId>org.jenkins-ci.tools</groupId>
+        <artifactId>maven-hpi-plugin</artifactId>
+        <extensions>true</extensions>
+        <configuration>
+          <archive>
+            <index>true</index>
+            <manifestEntries>
+              <Hudson-Version>1.609.18.1</Hudson-Version>
+              <Jenkins-Version>1.609.18.1</Jenkins-Version>
```

Comment:
problem with reflection is we must use the new call.  so it is not fine if it does not exist.
You would still need to set a baseline of jenkins to use.  so what would you use?
1.651.2 - can't plugin won;t be available for Cloudbees customers.
1.609.18.2 - can't plugin won't be buildable as we do not publish cloudbees propriatary wars...
1.609.3 - well will still blow up when the API is not available via reflection...

So you still will end up installing this plugin on something without the core update which will break the plugin. so given you need to barf we may as well just barf and keep code the same...